docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-kubernetes.md

```diff
@@ -0,0 +1,220 @@
+    identifier: software-kubernetes
+    parent: server-nodes-software
+    weight: 30
+type: docs
+---
+
+## Software versions
+
+The minimum version for a Kubernetes cluster and Helm chart are as follows:
+
+- Kubernetes 1.22
+- Helm 3.11.3
+
+The YugabyteDB Anywhere Helm chart has been tested using the following software versions:
+
+- Kubernetes - 1.22 to 1.25
+- Helm - 3.11.3
+
+For OpenShift clusters, the recommended OpenShift Container Platform (OCP) version is 4.6, with backward compatibility assumed but not guaranteed. In addition, ensure that you have the following:
+
+- The latest oc binary in your path. For more information, see [Installing the OpenShift CLI](https://docs.openshift.com/container-platform/4.6/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli).
+- A compatible kubectl binary in your path. See [Install and Set Up kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for more information, or create a kubectl symlink pointing to oc.
+
+## Specify ulimit and remember the location of core dumps
+
+The core dump collection in Kubernetes requires special care due to the fact that `core_pattern` is not isolated in cgroup drivers.
+
+You need to ensure that core dumps are enabled on the underlying Kubernetes node. Running the `ulimit -c` command within a Kubernetes pod or node must produce a large non-zero value or the `unlimited` value as an output. For more information, see [How to enable core dumps](https://www.ibm.com/support/pages/how-do-i-enable-core-dumps).
+
+To be able to locate your core dumps, you should be aware of the fact that the location to which core dumps are written depends on the sysctl `kernel.core_pattern` setting. For more information, see [Linux manual: core dump file](https://man7.org/linux/man-pages/man5/core.5.html#:~:text=Naming).
+
+To inspect the value of the sysctl within a Kubernetes pod or node, execute the following:
+
+```sh
+cat /proc/sys/kernel/core_pattern
+```
+
+If the value of `core_pattern` contains a `|` pipe symbol (for example, `|/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E` ), the core dump is being redirected to a specific collector on the underlying Kubernetes node, with the location depending on the exact collector. To be able to retrieve core dump files in case of a crash within the Kubernetes pod, it is important that you understand where these files are written.
+
+If the value of `core_pattern` is a literal path of the form `/var/tmp/core.%p`, no action is required on your part, as core dumps will be copied by the YugabyteDB node to the persistent volume directory `/mnt/disk0/cores` for future analysis.
+
+Note the following:
+
+- ulimits and sysctl are inherited from Kubernetes nodes and cannot be changed for an individual pod.
+- New Kubernetes nodes might be using [systemd-coredump](https://www.freedesktop.org/software/systemd/man/systemd-coredump.html) to manage core dumps on the node.
+
```

Comment:
- Is this step not required - https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes/#configure-storage-class. Compared it to the current preview docs page

Reply:
Was put under hardware